src/style.css

```diff
@@ -1,3 +1,75 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --active-color: #00acdc;
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should specify a generic family (e.g., 'sans-serif') as a fallback option to improve the robustness of your font styling.